e2e/tests/negative/upgrade_conflict_handle.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation    Manual Test Cases
+Test Tags    negative
+
+Resource    ../keywords/variables.resource
+Resource    ../keywords/common.resource
+Resource    ../keywords/longhorn.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/storageclass.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/backup.resource
+Resource    ../keywords/recurringjob.resource
+Library     DateTime
+
+Test Setup    Set up test environment
+Test Teardown    Cleanup test resources
+
+*** Keywords ***
+Get test start time
+    ${test_start_time}=    Get Current Date    result_format=datetime
+    Log    ${test_start_time}
+    Set Test Variable    ${test_start_time}
+
+*** Test Cases ***
+Test Stability of Longhorn with Large Workload
```

Comment:
If the only difference between these 2 test cases is whether to upgrade from the previous version, is there a way to reuse their test steps and simply pass an argument to control whether to upgrade?

Like:
```
Test Stability of Longhorn with Large Workload
    call_the_keyword    previous_version=

Test Upgrade Stability with Large Workload
    call_the_keyword    previous_version=${LONGHORN_STABLE_VERSION}
```

Reply:
Have a keywork `Perform recurring job workflow under load` to perform the workload operation, thank you.